utils/misc.js

```diff
@@ -43,13 +43,22 @@ export default class Miscellaneous {
         expressApp.enable('trust proxy');
 
         // add common data for response.
-        expressApp.use((_, res, next) => {
+        expressApp.use(async (_, res, next) => {
             // add project version info for all render pages.
             res.locals.version = ProjectConfigs.ghoslerVersion;
 
             // add no robots header tag to all.
             res.header('X-Robots-Tag', 'noindex, nofollow');
 
+            // provide ghosler.url to every view.
+            try {
+                const ghosler = await ProjectConfigs.ghosler();
+                res.locals.ghoslerUrl = ghosler.url || '';
+            } catch {
+                res.locals.ghoslerUrl = '';
+                logDebug(logTags.Express, 'Error loading ghosler.url config: ' + error.message);
+            }
```

Comment:
Why do we have a `try/catch` here? What kinda errors do you expect?
I'd suggest a `res.locals.ghoslerUrl = await ProjectConfigs.ghosler() || '';`, what do you think?

Reply:
yeah that should be fine. Just being cautious. Can update for ya without it no problem